ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/identity_provider_config_service.proto

```diff
@@ -0,0 +1,159 @@
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
+  // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
+  // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
+  rpc DeleteIdentityProviderConfig (DeleteIdentityProviderConfigRequest) returns (DeleteIdentityProviderConfigResponse);
+
+}
+
+message IdentityProviderConfig {
+  // The identity provider config identifier
+  // Must be a valid LedgerString (as describe in ``value.proto``).
+  // Required
+  string id = 1;
+
+  // When set, then the user identified by the identity provider is denied all access to the Ledger API.
+  // Otherwise, the user identified by the identity provider has access to the Ledger API as per the user's rights.
+  // Optional,
+  // Modifiable
+  bool is_deactivated = 2;
+
+  // Specifies the issuer of the JWT token that should be used in the "iss" claim of a JWT token to instruct the
+  // participant node to use this authentication domain to authenticate the token.
+  // The issuer value is a case sensitive URL using the https scheme that contains scheme, host,
+  // and optionally, port number and path components and no query or fragment components.
+  // Required
+  // Modifiable
+  string issuer = 3;
+
+  // The JWKS (JSON Web Key Set) URL.
+  // The Ledger API will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public
+  // key loaded from the given URL.
+  // Required
+  // Modifiable
+  string jwks_uri = 4;
+}
+
+
+message CreateIdentityProviderConfigRequest {
+  // Required
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message CreateIdentityProviderConfigResponse {
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message GetIdentityProviderConfigRequest {
+  // Required
+  string identity_provider_config_id = 1;
+}
+
+message GetIdentityProviderConfigResponse {
+  IdentityProviderConfig identity_provider_config = 1;
+}
+
+message ListIdentityProviderConfigsRequest {
+
+}
+
+message ListIdentityProviderConfigsResponse {
+  repeated IdentityProviderConfig identity_provider_configs = 1;
+}
+
+message UpdateIdentityProviderConfigRequest {
+  // The identity provider config to update.
+  // Required,
+  // Modifiable
+  IdentityProviderConfig identity_provider_config = 1;
+
+  // An update mask specifies how and which properties of the ``IdentityProviderConfig`` message are to be updated.
+  // An update mask consists of a set of update paths.
+  // A valid update path points to a field or a subfield relative to the ``IdentityProviderConfig`` message.
+  // A valid update mask must:
+  // (1) contain at least one update path,
+  // (2) contain only valid update paths.
+  // Fields that can be updated are marked as ``Modifiable``.
+  // An update path can also point to a non-``Modifiable`` fields such as 'id' and 'metadata.resource_version'
```

Comment:
Are you planning to to add `ObjectMeta` to the `IdentityProviderConfig` resource?

Reply:
Actually not... this is unlucky copy paste